### PR TITLE
Fix: Modal appears under the navbar #13195

### LINKED ISSUE #13195: Modal appears under the navbar
**Describe the bug**

The modal triggered by the three dots on the left of an article appears stuck under the navbar of the website, it is not readable entirely.

**To Reproduce**

After submitting an article but before publishing.
Click on the three dots on the top left of the article.
The modal will show up but a part of it is hidden behind the navbar

**Expected behavior**

The modal should appear entirely to see all the links.  It can appear under the navbar, or on top of it.

**Screenshots**

image

**Desktop (please complete the following information):**

- OS, version: MacOS Catalina 10.15.7
- Browser, version: Chrome Version 88.0.4324.192 (Official Build) (x86_64)

**Additional context**
button id="article-show-more-button"

### PR DESCRIPTION
**Pull request by veerreshr**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
fixes #13195 , where I made the modal appear in the downward direction by adding 2 classes to dropdown element

## QA Instructions, Screenshots, Recordings

Create a draft post and then click on 3 dots on the left(show more dropdown), now you can see dropdown appear in downward direction.
Tested on brave browser and mozilla firefox on ubuntu 20.04
Screenshot from 2021-07-15 04-15-39

### UI accessibility concerns?

No it wont effect user accessibility

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: As this is just a change in direction of dropdown, there wont be any test cases to test.
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: As this is just change in direction of dropdown, it wont effect others parts of code or UI.

Screenshots on different resolutions:
Screenshot from 2021-07-15 04-17-24
Screenshot from 2021-07-15 04-17-37
Screenshot from 2021-07-15 04-17-57

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -14,7 +14,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```
Comment (aitchiss) [inline db/schema.rb:17]: I think these changes are unintentional - would you mind removing them from this PR? thanks!

Comment (veerreshr) [reply db/schema.rb:17]: Yea sure those were auto generated while setting up. should I remove both the changes in this file?

Comment (aitchiss) [reply db/schema.rb:17]: Yes please! This happens sometimes when setting up locally (just a symptom of switching between different branches), and the changes can safely be discarded 👍

Comment (veerreshr) [reply db/schema.rb]: Done with the changes, could you review it again?

**Review by aitchiss — APPROVED** [review]
Looks good to me! Nice work 🚀

**Review by rhymes — APPROVED** [review]

**Pull request merged**